packages/storybook8/stories/Components/ControlBar/Buttons/Default/Default.story.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { ControlBarButton } from '@azure/communication-react';
+import { Airplane20Filled, VehicleBus20Filled, VehicleShip20Filled } from '@fluentui/react-icons';
+import React from 'react';
+
+const iconDict = {
+  airplane: <Airplane20Filled key={'airplaneIconKey'} primaryFill="currentColor" />,
+  bus: <VehicleBus20Filled key={'busIconKey'} primaryFill="currentColor" />,
+  ship: <VehicleShip20Filled key={'shipIconKey'} primaryFill="currentColor" />
+};
+
+const ControlBarButtonStory = (args: any): JSX.Element => {
+  const icon = iconDict[args.icons];
+
+  return <ControlBarButton {...args} onRenderIcon={() => icon} strings={{ label: args.icons }} labelKey={args.icons} />;
+};
+
+export const Default = ControlBarButtonStory.bind({});
```

Comment:
Do we want to leave this as `default` or should we call it by its component `ControlBar`

Reply:
I am following the naming convention from the storybook6, also this is for `default control bar button` so default makes sense. let me know if you think we should change it?
<img width="844" alt="Screenshot 2024-07-16 at 12 36 35 PM" src="https://github.com/user-attachments/assets/b208927c-40ed-4464-af0e-877d8bf8d674">